package assignments;

import java.util.Scanner;

public class Assignment_61 {
	
	//WAP to handle below 2 exceptions by using 2 catch blocks



	public static void main(String[] args) {
		/*try
		{
			Scanner S1 = new Scanner(System.in);*/
			int c = 1/0;
			//int a = S1.nextInt();
			System.out.println(c);
		
			/*catch(ArithmeticException a)
			{
				System.out.println("Arithmetic Exception");
			}
			catch(NullPointerException b)
			{
				System.out.println("NullPointer Exception ");
			}*/
		}

	}

	


